apps/dashboard/app/(app)/ratelimits/[namespaceId]/logs/components/table/hooks/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+
+let QUERY_TIMESTAMP = Date.now();
```

Comment:
bold move from Mr lintgod.

all caps suggests this is a constant, but it's not

Reply:
yeah. I wasn't sure about that one 😂